Common/OJS.Common/Constants/ServiceConstants.cs

```diff
@@ -4,13 +4,14 @@ public static class ServiceConstants
     {
         public static class CheckerTypes
         {
-            public const string ExactMatch = "exact-match";
-            public const string CaseInsensitive = "case-insensitive";
+            public const string ExactMatch = "exact";
+            public const string CaseInsensitive = "caseinsensitive";
             public const string Precision = "precision";
             public const string Sort = "sort";
             public const string Trim = "trim";
-            public const string TrimEnd = "trim-end";
-            public const string CSharpCode = "csharp-code";
+            public const string TrimEnd = "trimend";
+            public const string CSharpCode = "csharpcode";
```

Comment:
We should not modify existing names, in order to have compatibility with Legacy judge. Alpha workers will accept requests from it. Make sure checkers work, when `"CheckerType": "exact-match"` for example is passed to the request body, when submission is executed from the controller endpoint.